### PR TITLE
Item show page

### LINKED ISSUE #62: User Story 16, Item Show Page
```
As any kind of user on the system
When I visit an item's show page from the items catalog
My URI route is something like "/items/18"
I see all information for this item, including:
- the name of the item
- the description of the item
- a larger image of the item
- the merchant name who sells the item
- how many of the item the merchant has in stock
- the merchant's current price for the item
- an average amount of time it takes this merchant to fulfill this item

If I am a visitor or regular user, I also see a link to add this item to my cart
```

DO THESE STORIES FIRST:
- User Story 14, Items Index Page

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #65

child of #82

depends on #64

### PR DESCRIPTION
**Pull request by renecasco**
- [] Check this if the PR has been approved by the team to be a quick patch and the rest of this form will not be filled out

# Test Coverage Percentage (Model/Feature):
- rspec - 100%
- rspec spec/models - 100%

# What functionality does this accomplish?
Item show page

# What did you struggle on to complete?

# Implements/Fixes:
* description
closes #62 

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

# Please include an emoji of how you feel about this branch:
💯

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -7,4 +7,17 @@ class Item < ApplicationRecord
   belongs_to :user
   has_many :order_items
   has_many :orders, through: :order_items
+
+  def average_fulfilled_time
+    fulfillment = Item.joins(:orders)
```
Comment (milevy1) [inline app/models/item.rb]: Do we need to join :orders? If we are only using Item and order_items attributes, could we just use `Item.joins(:order_items)`
